concepts/variables/about.md

```diff
@@ -1 +1,86 @@
-# TODO
+# Introduction
+
+Recall that a `jq` program is a _pipeline of expressions_.
+Variable assigmnent follows this rule.
+A variable assignment is an expression that looks like:
+
+```jq
+... | expr as $varname | ...
+```
+
+## The filter outputs the input
+
+The input is used in the `expr` to produce a value that is bound to the variable name.
+The output of the variable assignment is the same as the input to the expression.
```

Comment:
Does this sentence mean that what goes into the front pipe is what comes out of the back pipe? I think it has to mean that, but it took me a while to get there. I'm wondering if there's a way to phrase it to help folks like me get there more quickly.